### ISSUE TITLE
[Spell] Blood Queen Lana'thel - Blood Mirror

### ISSUE THREAD

**Issue by hotdoggidy** [WoWManiaUK/Redemption#1266]
"Blood Mirror" (spell=71510) is not working as intended.

Blood Queen Lana'thel (npc=37955) is the final boss of the Blood quarter in ICC's Upper spire. When she is engaged she places a debuff on the tank called "Blood Mirror". This will link to the player nearest the tank and cause the linked player to receive 100% of damage taken by the tank. Optimal strategy is the Off-tank stacks **ON** the main tank, gets the link and takes the extra damage. If the OT moves too far from MT, the link **SHOULD** re-target the nearest player to the tank. 

Currently on Redemption, the link caused by "Blood Mirror" will **NOT** switch off its first linked target. The first player to be linked with the tank will take 100% of the damage taken by the tank for the entire fight. Even after death the link stays on the original target.

tl;dr: The link should always target (and re-target) the nearest player to the tank, regardless of role.

**Source**: http://wowwiki.wikia.com/wiki/Blood-Queen_Lana%27thel_(tactics)

"[Blood Mirror] 50,000 yd range—100% of damage done to you is dealt to the linked source. This affects the person closest to the tank."

Tank with link
![wowscrnshot_060717_145721](https://user-images.githubusercontent.com/23442126/26903471-3d63aa52-4b92-11e7-80c6-f33611034382.jpg)

OT with link (opposite side of room)
![wowscrnshot_060717_145722](https://user-images.githubusercontent.com/23442126/26903476-43434004-4b92-11e7-92e0-71492eabf15f.jpg)

OT dead (still with link)
![wowscrnshot_060717_145729](https://user-images.githubusercontent.com/23442126/26903488-4f99f320-4b92-11e7-8c9a-9c3c07d8b86f.jpg)

@tank - Link does not change targets
![wowscrnshot_060717_145741](https://user-images.githubusercontent.com/23442126/26903503-5899d4a4-4b92-11e7-8e26-74b47905f911.jpg)
